hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -157,6 +157,11 @@
       reset: "sys",
       reset_connections: {rst_main_ni: "sys", rst_fixed_ni: "sys_fixed"}
     },
+    { name: "peri",
+      clock_connections: {clk_peri_i: "fixed"},
+      reset: "sys_fixed",
```

Comment:
hmmm..i actually can't remember now what this reset is even needed more...shouldn't the `reset_connections` term below make this one obsolete?

Reply:
We can deal with this issue later :)